site/gatsby-site/i18n/locales/es/submit.json

```diff
@@ -0,0 +1,28 @@
+{
+  "New Incident Report": "Nuevo Reporte de Incidente",
+  "submitFormDescription": "El siguiente formulario creará un nuevo informe de incidentes para <2>revisión</2> e inclusión en la base de datos de incidentes de AI. Verifique cuidadosamente sus entradas para detectar problemas de contenido (por ejemplo, copiar y pegar accidentalmente anuncios). Para obtener detalles sobre el proceso de ingestión de la base de datos, consulte las <5>páginas de investigación</5> o <8>contáctenos si tiene preguntas.</8>",
```

Comment:
@cesarvarela I think we should document how the `<Trans>` component work on the README file https://github.com/cesarvarela/aiid/tree/feature-i18n-submit#ui-translations
Maybe a link to the component documentation is enough, especially for these `<2>`, `<5>` tags and how they are processed in the translation logic.

Reply:
I can infer that each number tag like `<2>` matches with a child HTML element of the element to translate, but it's better to have it explained.

Reply:
Agree let's track that in another issue https://github.com/responsible-ai-collaborative/aiid/issues/831